pageserver/src/tenant.rs

```diff
@@ -662,6 +668,25 @@ impl Tenant {
                 })?;
         }
 
+        // Walk through deleted timelines, resume deletion
```

Comment:
Thinking about attach -> fail -> restart sequence. On restart we wont list remote timelines so can miss this one because we havent created local metadata (any structure on disk). Not sure if its possible to fix without actual listing on startup, but I wont have time for full implement-test-merge cycle tomorrow.

Reply:
I think the right way would be to mention this in the issue, put it into backlog and link it in the source code. Objections?

Reply:
No objections.

Reply:
We'll need to do this before merging, otherwise it will be forgotten.

Reply:
Opened https://github.com/neondatabase/neon/issues/5043